astro/src/content/docs/get-started/use-cases/b2b2c.mdx

```diff
@@ -0,0 +1,190 @@
+
+Your customers are the circuses, and their users are *their* customers. Each circus has their own policies about login methods, password length, and more.
+
+You are super excited because you've landed both Ringling Bros. and Barnum & Bailey Circus and Cirque du Soleil as customers. They have customers of their own, who may be either individuals or organizations buying tickets. Here's a diagram of the relationship between ClownPass, its customers and the end users.
+
+<CircusesRelationships alt="Circus relationships diagram."/>
+
+The set of customers for each circus can overlap. Jamie Morgan is an example of a customer of both circuses.
+
+## Actors/Components
+
+* your customer and their client application (mobile app or browser)
+* your customer's user and their client application (mobile app or browser)
+* your application
+* FusionAuth
+
+## Implementation Steps
+
+There are three implementation steps to ensure your customers' users will always be able to get the tickets they need. This is a tenant based implementation, where users are stored in different FusionAuth tenants.
+
+The steps include:
+
+* Setting up your system to handle adding new customers
+* Letting customers' users sign in or register
+* Allowing your customers to manage their users and other settings
+
+Here's a high level sequence diagram of the process of adding new customers. Each customer is represented as a tenant.
+
+<AddNewCustomers alt="Adding new customers." />
+
+Here's a high level sequence diagram of a user logging in to one of the customer's ticketing systems. The application is responsible for mapping the user to the correct customer.
+
+<UserLoggingIn alt="Users logging in." />
+
+Here's a high level sequence diagram of a customer admin managing their users. The admin functions are integrated with the application admin screens using API calls.
+
+<AdminManagingUsers alt="Tenant admin management of their users." />
+
+Let's look at these steps in detail.
+
+### Adding New Customers
+
+To add new customers, you need to make changes to both the ClownPass ticketing application and FusionAuth configuration.
+
+Make sure your circus SaaS application can handle multiple tenants. At a minimum, you need to ensure your application has some way to distinguish each customer's access points. There are a couple of ways to do this, but the most typical is to offer a different hostname for each customer. If the ticketing application is hosted at `example.com`, you might have the following hostnames:
+
+* `ringling.example.com` for Ringling Bros. and Barnum & Bailey Circus ticketing
+* `cirquedesoleil.example.com` for Cirque du Soleil ticketing
+
+Ensure your web application can respond to multiple hostnames. If you don't want to have customer specific hostnames, there are [other options](/docs/extend/examples/multi-tenant#data-plane-application-tenant-determination).
```

Comment:
This is different than custom domains for fusionauth instances. This is custom domains for the application (the ticketing app in this case) so that users can be routed appropriately to the correct FusionAuth application based on the hostname.

Reply:
Understood, but let's say there is thescarycircus.com, thehappycircus.com, thegroovycircus.com. We describe how your customers can customize and white label their login experience, so they do that, then they change all of their email templates, so on and so forth. But login will still happen at the parent auth companies domain. In some cases that may be okay, but in other cases it may not be.

Reply:
With that, I am also not an authority on the matter. I understood that you were describing non-FusionAuth domain names. The suggestion wasn't a correction as much of a suggestion for a possible "in addition to" for a more pure white labeling.

Reply:
Thanks for explaining your concerns! I'll see if I can work it in somehow.